psi4/driver/driver_cbs.py

```diff
@@ -278,21 +444,25 @@ def scf_xtpl_helgaker_3(functionname, zLO, valueLO, zMD, valueMD, zHI, valueHI,
 
 
 #def corl_xtpl_helgaker_2(functionname, valueSCF, zLO, valueLO, zHI, valueHI, verbose=True):
-def corl_xtpl_helgaker_2(functionname, zLO, valueLO, zHI, valueHI, verbose=True):
+def corl_xtpl_helgaker_2(functionname, zLO, valueLO, zHI, valueHI, verbose=True, alpha=None):
```

Comment:
Can we do a `alpha=3` and indicate this default in the description. Using the workaround below is only really useful for non-atomics like lists.

Reply:
Just to be clear, you mean this for all functions, not just the one above?

Reply:
Ah, sorry I didn't know you were around so I fixed them directly. Please review the following: https://github.com/psi4/psi4/pull/963/commits/948cc7d4d7eddb5434715a12b34deb9840ccd215

Reply:
Your comment makes sense, @dgasmith, but I think it doesn't work b/c `alpha` is always passed, so the `None` is needed to indicate no user involvement.

`hiloargs = {'alpha': cbs_alpha[stage['d_stage']]}`

```
cbs_alpha = {
+        'scf': kwargs.get('cbs_scf_alpha', kwargs.get('scf_alpha', None)),
+        'corl': kwargs.get('cbs_corl_alpha', kwargs.get('corl_alpha', None)),
+        'delta': kwargs.get('cbs_delta_alpha', kwargs.get('delta_alpha', None)),
+        'delta2': kwargs.get('cbs_delta2_alpha', kwargs.get('delta2_alpha', None)),
+    }
```